src/app/chart/donut/donut-config.ts

```diff
@@ -0,0 +1,36 @@
+import { ChartConfig } from '../chart-config';
+/**
+ * A config containing properties for the sparkline chart
+ */
+export class DonutConfig extends ChartConfig {
+
+  /**
+   * An optional function to customize the text of the center label
+   */
+  centerLabelFn?: any;
+
+  /**
+   * An optional function to handle when donut is clicked
+   */
+  onClickFn?: any;
+
+  /**
+   * The height of the donut chart
+   */
+  chartHeight?: number;
+
+  /**
+   * C3 inherited configuration for colors
+   */
+  colors?: any;
```

Comment:
It's not clear what "colors" is. Please add an example to the comments? For example:

'colors' : {
&nbsp;&nbsp;'Cats': '#0088ce',     // blue
&nbsp;&nbsp;'Hamsters': '#3f9c35', // green
&nbsp;&nbsp;'Fish': '#ec7a08',     // orange
&nbsp;&nbsp;'Dogs': '#cc0000'      // red
}

Reply:
I've updated the comment with an example. I'm reluctant to add a typed object for c3 inherited configuration as it mimics c3 configuration exactly. Maybe a link to the c3 API that it's intended to match would work for all c3 inherited configuration?